pkg/convert/pot_test.go

```diff
@@ -0,0 +1,317 @@
+package convert
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func Test_ToPot(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name     string
+		input    model.Messages
+		expected []byte
+	}{
+		{
+			name: "When all values are provided",
+			input: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello, world!",
+						Message:     "Bonjour le monde!",
+						Description: "A simple greeting",
+						Fuzzy:       true,
+					},
+					{
+						ID:          "Goodbye!",
+						Message:     "Au revoir!",
+						Description: "A farewell",
+						Fuzzy:       true,
+					},
+				},
+			},
+			expected: []byte(`"Language: en
+#. A simple greeting
+#, fuzzy
+msgid "Hello, world!"
+msgstr "Bonjour le monde!"
+#. A farewell
+#, fuzzy
+msgid "Goodbye!"
+msgstr "Au revoir!"
+`),
+		},
+		{
+			name: "When msgid is multiline",
+			input: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello, world!\\n\"\n\"very long string\\n",
```

Comment:
Is it supposed to be the following for msgid? 

```po
msgid "" 
"Hello, world!\n"
"very long string\n"
```

```suggestion
						ID:          `Hello, world!
very long string
`,
```

Reply:
If I correctly understood..then yes ![image](https://user-images.githubusercontent.com/94634373/221526750-526a5475-878b-4992-8b40-485da2b4ea74.png)

Reply:
The original text does not contain `"`. We normalize the value

```pot
msgid ""
"Hello, world!\n"
"very long string\n"
```
to

```
Hello, world!
very long string
```

```go
x := "Hello, world!\nvery long string\n"
```